src/sequelize/db.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+const { Sequelize } = require('sequelize');
+
+const sequelize = new Sequelize('postgres', 'postgres', 'oleh', {
```

Comment:
better to create constants
```suggestion
const DB_NAME = 'postgres';
//...

const sequelize = new Sequelize(DB_NAME, 'postgres', 'oleh', {
```